wire-ios-share-engine/Sources/SharingSession.swift

```diff
@@ -440,6 +459,17 @@ public final class SharingSession {
             mlsActionExecutor: mlsActionExecutor
         )
 
+        let mlsService = MLSService(
+            context: coreDataStack.syncContext,
+            notificationContext: coreDataStack.syncContext.notificationContext,
+            coreCryptoProvider: coreCryptoProvider,
+            conversationEventProcessor: ConversationEventProcessor(context: coreDataStack.syncContext),
+            featureRepository: FeatureRepository(context: coreDataStack.syncContext),
+            userDefaults: .standard,
```

Comment:
Not related directly to this PR, but it looks weird that we use .standard. 

Looking at the code, it's used to store the date of  keyPackageQueriedTime:
```
 userDefaults.set(Date(), forKey: .keyPackageQueriedTime)
 ```
 
 To me it makes sense that this value is common to all processes (NSE, SharingExt, mainApp

```suggestion
            userDefaults: .shared(),
```
or .applicationGroup

Note: .shared() and .applicationGroup seems to be the same code but not defined at the same level.

Reply:
Are you sure ? Because looking at the MLSService initialization in `ZMUserSessionBuilder` part of WireSyncEngine, we use the .standard UserDefaults

Reply:
I think this should be investigated. My understanding is that `.standard` will be a different store in the main app and the share extension. I assume it is important that the same store is used.

Reply:
Update: I think you're right, we cannot use the .standard to share data between a host app and its app extension (as also mentioned by Sam)

Reply:
as discussed with @netbe, we can keep the .standard one as this value doesn't need to be shared across targets
@samwyndham